index.js

```diff
@@ -317,6 +316,9 @@ function writeFile(file, data, options) {
 
 function trimSourceMap(molder) {
   var map = molder.sourcemap;
+  if(!map) {
```

Comment:
For consistency with the existing coding style this should look like this:

```js
  if (map === undefined)
    return '';
```

(or `null`, don't know which it is in this case.)